meta-dts-distro/recipes-tests/converged-security-suite/bg-suite_2.7.0.bb

```diff
@@ -0,0 +1,21 @@
+FILESEXTRAPATHS += "converged-security-suite/files"
```

Comment:
@DaniilKl it is better to place file in `bg-suite` directory and then use `FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"`

Reply:
@TomaszAIR Amended [here](https://github.com/Dasharo/meta-dts/pull/49/commits/265a33ac76bb377b166ca89810e2ce0e066f55cd)